java/arcs/core/storage/DirectStore.kt

```diff
@@ -205,7 +205,7 @@ class DirectStore<Data : CrdtData, Op : CrdtOperation, T> /* internal */ constru
       }
     }.also {
       log.verbose { "Model after proxy message: ${localModel.data}" }
-      devTools?.onDirectStoreProxyMessage(proxyMessage = message)
+      devTools?.onDirectStoreProxyMessage(proxyMessage = message as UntypedProxyMessage)
```

Comment:
Here, and in ReferenceModeStore, I've had to cast to UntypedProxyMessage whereas before when we were using ProxyMessage<*, *, *> casting wasn't necessary. Do you think it's still better to use UntypedProxyMessage here?

Reply:
Yeah I think this is ok. The cast should always be valid